tracing/src/test/java/org/axonframework/extensions/tracing/TracingCommandGatewayTest.java

```diff
@@ -69,6 +67,9 @@ public void testSendWithCallback() {
         List<MockSpan> mockSpans = mockTracer.finishedSpans();
         assertThat(mockSpans.size(), is(1));
         assertThat(mockSpans.get(0).operationName(), is("sendCommandMessage"));
+
+        mockTracer.scopeManager().active().close();
```

Comment:
Sorry, what is it you are asserting here? I am not familiar with opentracing test library, but it looks like you are calling a method on a mock object?

Also, your PR states it does not change any behavior, so it is not expected to see changes in tests.